src/app/filters/filter-query.ts

```diff
@@ -7,4 +7,6 @@
 export class FilterQuery {
   id?: string;
   value: string;
+  imageUrl?: string;
+  iconClass?: string;
```

Comment:
I'd still like to see this work in the demo. It's just a URL and class name, right?